lib/set-user.go

```diff
@@ -24,8 +24,19 @@ import (
 )
 
 // SetUser sets the user the pod will run as in the untarred ACI stored at
-// acipath.
-func SetUser(acipath, user string) error {
+// a.CurrentACIPath.
+func (a *ACBuild) SetUser(user string) (err error) {
+	err = a.getLock()
```

Comment:
Two nits:
1) now that it's no longer returning anything, s/getLock/lock/g and s/releaseLock/unlock/g
2) abbreviate these constructions everywhere:
```
if err = a.lock(); err != nil {
    return
}
defer func() {
  if err1 := a.releaseLock(); err1 != nil {
    err = err1
}()
```